scripts/deploy.sh

```
@@ -0,0 +1,85 @@
+  fi
+  sleep $SLEEP_SEC
+done
+
+log "❌ 헬스체크 실패. 이전 태그로 롤백."
+log "  롤백 APP_TAG=$PREV_APP_TAG FRONTEND_TAG=$PREV_FRONTEND_TAG"
+export APP_TAG=$PREV_APP_TAG
+export FRONTEND_TAG=$PREV_FRONTEND_TAG
+docker compose up -d --no-deps --no-build app frontend
+
+log "롤백 후 헬스체크 재확인"
+for i in $(seq 1 15); do
```

Shellcheck (0.11.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)